### DISCUSSION TITLE
CooperCasino 0.0.1a

### DISCUSSION THREAD

**Discussion by secureshellprotocol** [ECE-366-Final-Project/Back-End#41] [Announcements]
## Alpha 1 release
**ALL API IS SUBJECT TO CHANGE**  
We will generate a release.yml later  
We implemented a working slots game, and a self-deploying database which auto-populates required information (eg, slot payouts, user table, etc). Please see the README.md  

Next version will be prioritizing a standardized API with general codes, in order to remove redundant code. We also will be planning to implement stricter unit testing, including a framework to automatically tell us how much our code is covered.  

<hr /><em>This discussion was created from the release <a href='https://github.com/ECE-366-Final-Project/Back-End/releases/tag/alpha'>CooperCasino 0.0.1a</a>.</em>

**Comment by evanrflof** [comment]
yippie
